### PR TITLE
Vector 1

### PR DESCRIPTION
**Pull request by doqhne**
Changed tick marks to match NCL plot and changed colorbar to match NCL plot.

Closes issue #440

Previous version:
previous_version

New version: 
new_version

Original NCL plot:
NCL_original

### CODE REVIEW

File: Gallery/Vectors/NCL_vector_1.py [hunk 1]
```diff
@@ -72,80 +95,68 @@
...
-
-# Add color bar
+# Add minor tick marks
+gv.add_major_minor_ticks(ax,
+                         x_minor_per_major=4,
+                         y_minor_per_major=4,
+                         labelsize=14)
+
+# Add and customize colorbar
 cbar_ticks = np.arange(24, 29.1, .3)
```
Comment (michaelavs) [inline Gallery/Vectors/NCL_vector_1.py:131]:
```suggestion
cbar_ticks = np.arange(24, 29, .3)
```
I know you didn't edit this part, but for some reason the endpoint of this range was being included in the colorbar. Just switching it to 29 will resolve this.

**Review by michaelavs — COMMENTED** [review]
Looks good overall, just a couple of comments to be addressed!

**Comment by anissa111** [comment]
A couple points:

On my side, the background for the label is not showing up. Also, if you look at the screenshot below, it seems like we might be missing a level above 28.8, which would also make sense because it seems like the original plot has a section of darker purple that the recreation does not. 

Also, we might want to consider switching colormaps to reflect best practices as in the other examples

image

**Comment by doqhne** [comment]
I could not get the color bar edges to extend to the end of the colorbar, but the levels on the actual plot are correct now.

**Review by anissa111 — APPROVED** [review]
Looks good!!

**Pull request merged**